WaterLog_Backend/Controllers/LocationsController.cs

```diff
@@ -0,0 +1,88 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Storage;
+using Microsoft.Extensions.Configuration;
+using WaterLog_Backend.Models;
+
+namespace WaterLog_Backend.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class LocationsController : ControllerBase
+    {
+        private readonly DatabaseContext _db;
+        readonly IConfiguration _config;
+        public LocationsController(DatabaseContext context, IConfiguration config)
+        {
+            _db = context;
+            _config = config;
+        }
+
+        // GET api/values
+        [HttpGet]
+        public async Task<ActionResult<IEnumerable<LocationsEntry>>> Get()
```

Comment:
Get? Get what? Have a more descriptive function name

Reply:
Here you getting values of the Location api. I changed the name to be GetLocations